### PR TITLE
[FR] Beatmap_Editor

### PR DESCRIPTION
**Pull request by DoryanR**

### CODE REVIEW

File: wiki/Beatmap_Editor/fr.md [hunk 1]
```diff
@@ -1,18 +1,32 @@
-# Éditeur de Beatmap
+# L'éditeur de Beatmap
 
-![Le mode Compose](Compose/Compose.png "Le mode Compose")
+**L'éditeur de beatmap** est l'interface graphique intégrée au jeu d'osu! Pour la création de [beatmaps](/wiki/Beatmaps).
```
Comment (Pachiru) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion
**L'éditeur de beatmap** est une interface graphique intégrée au jeu pour la création de [beatmaps](/wiki/Beatmaps).
```

File: wiki/Beatmap_Editor/fr.md [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+Il est séparé en quatre parties:
 
- **L'éditeur de beatmap** est l'interface graphique d'osu! pour créer des [beatmaps](/wiki/Beatmaps). Il est divisé en 4 modes :
+- [Compose](Compose) : L'endroit où la majorité des beatmapping sont créer.
```
Comment (Pachiru) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion
- [Compose](Compose) : L'endroit où se déroule la majorité du processus de création de beatmap.
```

File: wiki/Beatmap_Editor/fr.md [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
 
--   [Compose](/wiki/Beatmap_Editor/Compose) : c'est ici que les [objets](/wiki/Hit_Objects) sont placés et déplacés. C'est dans ce mode que les mappeurs passent la plupart de leur temps.
--   [Design](/wiki/Beatmap_Editor/Design) : c'est ici que le storyboard est créé. La plupart des mappers n'utilisent que le bouton « insertion d'un fond ».
--   [Timing](/wiki/Beatmap_Editor/Timing) : c'est ici que l'offset et le BPM de la musique sont spécifiés. Cette fenêtre contrôle aussi les hitsounds s'ils ne sont pas constants durant la map.
--   [Song Setup](/wiki/Beatmap_Editor/Song_Setup) : c'est ici que les diverses options de la map sont spécifiées. Ceci inclut la couleur des combos et la configuration de la difficulté.
+- [Design](Design) : L'endroit pour la création de storyboards. Cependant, la plupart des mappeurs n'utiliseront que l'option "Image de fond/Vidéo" ici.
 
-## Articles liés
+- [Timing](Timing) : L'endroit où toutes les nouvelles beatmaps sont créer. Ceci contrôle le BPM (Battement Par Minute) et d'autres options de hitsounds.
```
Comment (Pachiru) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion
- [Timing](Timing) : La première étape pour la création d'une beatmap. Ceci permet de contrôler le BPM (Battement Par Minute) et d'autres options de hitsounds.
```

File: wiki/Beatmap_Editor/fr.md [hunk 4]
```diff
@@ -1,18 +1,32 @@
...
 
--   [Bruitages](/wiki/Beatmap_Editor_Guides/Custom_Sample_Overrides)
--   [Éditeur de beatmaps osu!mania](/wiki/Game_Modes/osu!mania)
--   [Espacement des notes](/wiki/Beatmap_Editor/Distance_Snap)
--   [Kiai Time](/wiki/Beatmap_Editor/Kiai_Time)
--   [Représentation du tempo](/wiki/Beatmap_Editor/Beat_Snap_Divisor)
+- [Song Setup](Song_Setup) : L'endroit où toutes les nouvelles beatmaps sont créer. Ceci définit les valeurs concernant le titre, l'artiste, le nom de la version, la difficulté, les couleurs de combo, etc.
```
Comment (Pachiru) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion
- [Song Setup](Song_Setup) : Ceci définit les valeurs concernant le titre, l'artiste, le nom de la version, la difficulté, les couleurs de combo, etc.
```

File: wiki/Beatmap_Editor/fr.md [hunk 5]
```diff
@@ -1,18 +1,32 @@
...
+
+D'autres composants de l'éditeur de beatmap incluent :
+
+- [AiMod](AiMod) : Le logiciel que tous les beatmappers peuvent utiliser.
```
Comment (Pachiru) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion
- [AiMod](AiMod) : Fonctionnalité qui permet aux beatmappeurs d'afficher des potentielles erreurs sur la beatmap.
```

File: wiki/Beatmap_Editor/fr.md [hunk 6]
```diff
@@ -1,18 +1,32 @@
...
+
+- [Beat Snap Divisor](Beat_Snap_Divisor) : Le logiciel sur la chronologie qui vous aide à faire une beatmap, en termes de temps.
```
Comment (Pachiru) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion
- [Beat Snap Divisor](Beat_Snap_Divisor) : Un outil qui permet de changer le rythme des battements.
```

File: wiki/Beatmap_Editor/fr.md [hunk 7]
```diff
@@ -1,18 +1,32 @@
...
+
+- [Distance Snap](Distance_Snap) : Le logiciel qui affecte la proximité ou la distance des objets touchés.
+
+- [Kiai Time](Kiai_Time) : Notes sur les fontaines en étoile et les lumières flashy pendant une certaine section d'une beatmap.
```
Comment (Pachiru) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion
- [Kiai Time](Kiai_Time) : Permet d'ajouter une section Kiai Time, qui est souvent placée lors d'un moment fort de la musique. Le Kiai Time rajoute des petits effets en jeu, comme des effets d'étoiles sur les cotés de l'écran, des clignotements plus intenses des cercles, etc...
```

File: wiki/Beatmap_Editor/fr.md [hunk 8]
```diff
@@ -1,18 +1,32 @@
...
+
+- [Menu](Menu) : Une explication sur les boutons de menu en haut de l'éditeur de beatmap.
+
+- [SB Load](SB_Load) : Un logiciel utilisé pour dire à quel point un storyboard est dégradant pour les performances. Tout nombre supérieur à 5x est mauvais pour les machines anciennes.
+
+- [Slider](Slider) : Une explication de base du fonctionnement des sliders dans osu!
+
+- [Timelines](Timelines) : La partie supérieure de l'éditeur beatmap qui affiche les objets frappés placés, les ruptures, etc.
```
Comment (Pachiru) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion
- [Timelines](Timelines) : La partie supérieure de l'éditeur beatmap qui affiche les objets placés, les pauses, etc.
```

File: wiki/Beatmap_Editor/fr.md [hunk 9]
```diff
@@ -1,18 +1,32 @@
-# Éditeur de Beatmap
+# L'éditeur de Beatmap
 
-![Le mode Compose](Compose/Compose.png "Le mode Compose")
+**L'éditeur de beatmap** est une interface graphique intégrée au jeu pour la création de [beatmaps](/wiki/Beatmaps).
+Il est séparé en quatre parties:
 
- **L'éditeur de beatmap** est l'interface graphique d'osu! pour créer des [beatmaps](/wiki/Beatmaps). Il est divisé en 4 modes :
+- [Compose](Compose) : L'endroit où se déroule la majorité du processus de création de beatmap.
```
Comment (cl8n) [inline wiki/Beatmap_Editor/fr.md]: there should be no empty lines between list items (this makes them render double-spaced)

File: wiki/Beatmap_Editor/fr.md [hunk 10]
```diff
@@ -1,18 +1,32 @@
...
```
Comment (cl8n) [inline wiki/Beatmap_Editor/fr.md]: remove the space before `:` (also wherever else this appears)

File: wiki/Beatmap_Editor/fr.md [hunk 11]
```diff
@@ -1,18 +1,32 @@
-# Éditeur de Beatmap
+# L'éditeur de Beatmap
 
-![Le mode Compose](Compose/Compose.png "Le mode Compose")
+**L'éditeur de beatmap** est une interface graphique intégrée au jeu pour la création de [beatmaps](/wiki/Beatmaps).
+Il est séparé en quatre parties:
```
Comment (cl8n) [inline wiki/Beatmap_Editor/fr.md]:
```suggestion

Il est séparé en quatre parties:
```

Comment (cl8n) [reply wiki/Beatmap_Editor/fr.md]: would appreciate if u can make this change to en.md as well

File: wiki/Beatmap_Editor/fr.md (hunk 10 above)
Comment (cl8n) [reply wiki/Beatmap_Editor/fr.md]: nevermind about this one, I didn't realise it's standard in French to do this

**Pull request merged**